### PR TITLE
doc,test(3d-tiles): glTF content extension

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/3d-tiles/docs/api-reference/tiles-3d-loader.md [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
-| Decoder Type          | Asynchronous                                                                                                       |
-| Worker Thread Support | No                                                                                                                 |
-| Streaming Support     | No \*                                                                                                              |
-| Subloaders            | `DracoLoader` (`.pnts`), `GLTFLoader` (`.b3dm`, `.i3dm`), `ImageLoader` (.`.jpg`, `.png`), `TextureLoader` (`.ktx`)                                                           |
+| Loader                | Characteristic                                                                                                      |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| File Extensions       | `.b3dm`,`.i3dm`, `.pnts`, `.cmpt`                                                                                   |
```
Comment (ibgreen) [inline modules/3d-tiles/docs/api-reference/tiles-3d-loader.md:11]: Are we adding glb support here?
```suggestion
| File Extensions       | `.b3dm`,`.i3dm`, `.pnts`, `.cmpt`, `.glb`, `.gltf`                                                                                   |
```

File: modules/3d-tiles/docs/api-reference/tiles-3d-loader.md [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
+| File Type             | Binary (with linked assets)                                                                                         |
+| File Format           | [3D Tiles](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#tile-format-specifications)  |
+| Data Format           | [Data Formats](#data-formats)                                                                                       |
+| Decoder Type          | Asynchronous                                                                                                        |
+| Worker Thread Support | No                                                                                                                  |
+| Streaming Support     | No \*                                                                                                               |
+| Subloaders            | `DracoLoader` (`.pnts`), `GLTFLoader` (`.b3dm`, `.i3dm`), `ImageLoader` (.`.jpg`, `.png`), `TextureLoader` (`.ktx`) |
```
Comment (ibgreen) [inline modules/3d-tiles/docs/api-reference/tiles-3d-loader.md:18]:
```suggestion
| Subloaders            | `DracoLoader` (`.pnts`), `GLTFLoader` (`.b3dm`, `.i3dm`, `.glb`, `.gltf`), `ImageLoader` (.`.jpg`, `.png`), `TextureLoader` (`.ktx`) |
```

**Review by ibgreen — APPROVED** [review]

**Review by ibgreen — APPROVED** [review]
- Should we update what's new?
- Should we specify with a badge that GLB/GLTF/Basis support is starting from v3.1?

File: modules/3d-tiles/docs/api-reference/tiles-3d-loader.md (hunk 1 above)
Comment (ibgreen) [reply modules/3d-tiles/docs/api-reference/tiles-3d-loader.md:11]: Let's follow the spec. if the spec only support glb let's do that, if the spec supports both lets aim for that.

**Comment by belom88** [comment]
> * Should we update what's new?
> * Should we specify with a badge that GLB/GLTF/Basis support is starting from v3.1?

Ok. I will do it

**Pull request merged**